.evergreen/config.yml

```diff
@@ -2155,7 +2158,11 @@ buildvariants:
 
   - matrix_name: "load-balancer-test"
     # The LB software is only available on Ubuntu 18.04, so we don't test on all OSes.
-    matrix_spec: { version: ["5.0", "latest"], os-ssl-40: ["ubuntu1804-64-go-1-16"] }
+    # The new "loadBalancerPort" option is supported starting with server 5.2, which responds
+    # correctly to "hello" commands with a service ID when behind a load balancer. Only run load
+    # balancer tests on server 5.2+.
+    # TODO(GODRIVER-2292): Add version "5.2" when it's available on Evergreen.
```

Comment:
You may be able to remove this TODO. Alternatively, add `rapid` to the `version` list.

From this [slack conversation](https://mongodb.slack.com/archives/C72LB5RPV/p1644016702267779) I interpret the intent to be that drivers to test against the following set of server versions:
- Supported major releases (for Go: `2.6`, `3.0`, `3.4`, `3.6`, `4.0`, `4.2`, `4.4`, `5.0`)
- The latest build from evergreen (e.g. `5.3.0-alpha...`)
- The latest rapid release (currently `5.2` only, not `5.1`).

Reply:
I agree. It sounds like ideally we'd want all Evergreen tasks that test against `latest` to probably also test against `rapid`. You could separate that work into another ticket and leave a relevant `TODO` here if that seems overboard for this PR.

Reply:
Ah, I didn't know that existed. It's weird that `"rapid"` refers to version 5.2, but using the literal version string `"5.2"` results in a configuration error. At any rate, that sounds like a good change, so I'll add `"rapid"` to the list.

I'll also create a ticket to update all tasks that use `"latest"` to also use `"rapid"`.